core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -75,7 +75,7 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
 
   override val rpcEnv: RpcEnv = sc.env.rpcEnv
 
-  private[spark] var scheduler: TaskScheduler = null
+  private[spark] var taskScheduler: TaskScheduler = null
```

Comment:
Thank you for making a PR, but I don't think we need this change inside `HeartbeatReceiver`.

Reply:
Sure @dongjoon-hyun. I am curious to understand why you suggest it is not needed ?